tests/unit/src/main/scala/scala/meta/internal/metals/debug/TestDebugger.scala

```diff
@@ -103,7 +103,7 @@ final class TestDebugger(
   def shutdown: Future[Unit] = {
     Debug.printEnclosing()
     for {
-      _ <- terminated.future
+      _ <- terminated.future.withTimeout(60, TimeUnit.SECONDS)
```

Comment:
If we don't terminate within 60 seconds most likely terminate event will never arrive. This should make the DAP suites not wait for 10 minutes.

Reply:
60s here, 60s below which adds up to 120. Will 30 for each do the work?

Reply:
Probably, but I don't think both would happen at the same time (hopefully)